packages/thumbnail-grid-block/src/ThumbnailGridBlock.tsx

```diff
@@ -0,0 +1,230 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
sort imports

-- external

-- frontify

-- internal

Reply:
Are this best practices at Frontify? I usually let vscode sort them alphabetically.